.changeset/beige-pumpkins-brush.md

```diff
@@ -0,0 +1,7 @@
+---
+"@justeattakeaway/pie-icons-react": minor
+"@justeattakeaway/pie-icons-vue": minor
+"@justeattakeaway/pie-icons": minor
```

Comment:
is renaming an icon a breaking change for existing consumers?

Reply:
Usually it would be, but this package isn't used anywhere yet, so I've taken a bit of a liberty with it and just minor bumped it